superset/utils/pandas_postprocessing.py

```diff
@@ -916,3 +916,29 @@ def outliers(series: Series) -> Set[float]:
         for metric in metrics
     }
     return aggregate(df, groupby=groupby, aggregates=aggregates)
+
+
+def resample(
+    df: DataFrame,
+    resample_rule: str,
+    resample_method: str,
+    time_column: str,
+    resample_fill_zero: bool = False,
```

Comment:
To futureproof this, would it make sense to have a `fill_value: Optional[Union[float, int]]` here?